tests/optimizers/test_binary.py

```diff
@@ -28,3 +29,24 @@ def optimizer_reset(self, options):
         opt.optimize(sphere, 10)
         opt.reset()
         return opt
+
+    def test_binary_correct_pos(self, options):
```

Comment:
It may be easier to check whether it found the best by using the [history](https://github.com/ljvmiranda921/pyswarms/blob/master/pyswarms/discrete/binary.py#L183) of the optimizer, what do you think?

Reply:
I think calculating the cost from the position is a good test to have. I also agree with your earlier comment; a need to test that the optimise method indeed returns the best cost? I'll have a look into that.

Thanks for the comments @whzup  :+1:

Reply:
I think I understand what you meant now.

Reply:
@ichbinjakes Yes, excuse me, I may have not expressed myself clearly enough 😕:smile: